src/components/AppHeader/MobileMenu/BackButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { LegacyChevronLeft1pxIcon } from '@deriv/quill-icons';
+import { Text } from '@deriv-com/ui';
+
+type TBackButton = {
+    buttonText: string;
+    onClick: () => void;
+};
+
+export const BackButton = ({ buttonText, onClick }: TBackButton) => (
+    <button className='flex items-center w-full pt-[2rem] p-[3.2rem]' onClick={onClick}>
```

Comment:
can we not use the button from deriv-com/ui?

Reply:
![Screenshot 2024-06-13 at 12 24 24 PM](https://github.com/deriv-com/p2p/assets/93518187/b10fae68-ea98-483e-b500-c7ce4dc4aaf4)

It is this component that's why I didn't use button component from UI
IMO we should not use button component in this case

Reply:
ah.. got it..